tests/cpp-tests/Classes/AppDelegate.cpp

```diff
@@ -73,14 +76,14 @@ bool AppDelegate::applicationDidFinishLaunching()
     director->setAnimationInterval(1.0f / 60);
 
     auto screenSize = glview->getFrameSize();
-    auto designSize = Size(480, 320);
+    auto designSize = Size(1024/2, 2112/2);
 
     auto fileUtils = FileUtils::getInstance();
     std::vector<std::string> searchPaths;
     
     if (screenSize.height > 320)
     {
-        auto resourceSize = Size(960, 640);
+        auto resourceSize = Size(1024, 2112);
```

Comment:
Why change the default value of design resolution size and resource size?
The design resolution and content scale factor should not be changed since they were confirmed when write the test case.

Reply:
reverted